src/accessibility/AccessibilitySystem.ts

```diff
@@ -21,7 +21,7 @@ import type { isMobileResult } from '../utils/browser/isMobile';
  * This module is a mixin for {@link AbstractRenderer} and will need to be imported if you are managing your own renderer.
  * Usage:
  * ```js
- * import 'pixi.js/accessibility';
+ * import 'pixi.js';
```

Comment:
the `pixi.js/accessibility` import does exist and should be used if you are managing your own renderer
so this shouldn't be removed

Reply:
Ah good point thanks, I was coming at it from more of a general "how to make something accessible" angle.

I've worked it a little more so it separates those two concepts, ie 1) How to include if building your own renderer 2) How to make something accessible.